app/config/rules/bs_golden.json

```diff
@@ -2,6 +2,12 @@
 	"title": "BeeStation Rules",
 	"description": "",
 	"rules": [
+			"title": "Admin Discretion",
+			"clauses": [
+				"Admins have the authority to 'ignore' rules when they feel it's in the best interest of the server/playerbase. They will, however, be fully accountable for their actions.",
+				"Listen to them without argument during the round, but if you feel there is a problem with their actions, report them on the forums afterwards. The Admin Conduct can be found here: https://pastebin.com/dpY4CyJB"
```

Comment:
Missed this in my initial review. A pastebin link? Really? Can we not have a forum thread or something?

Reply:
A wiki article that has restricted editing might actually be better